margins.go

```diff
@@ -62,3 +74,38 @@ func (c *Client) GetOrderMargins(orderParams []OrderMarginParam) ([]OrderMargins
 
 	return out, nil
 }
+
+func (c *Client) GetBasketMargins(orderParams []OrderMarginParam, considerPositions bool, compactMode bool) (BaksetMargins, error) {
+	body, err := json.Marshal(orderParams)
+	if err != nil {
+		return BaksetMargins{}, err
+	}
+
+	var headers http.Header = map[string][]string{}
+	headers.Add("Content-Type", "application/json")
+
+	uri := URIBasketMargins
+	if compactMode || considerPositions {
```

Comment:
On a second thought, I guess we will have to use `net/url` to form the url query params

```go
v := url.Values

if gp.Compact { v.Set("compact", "true") }
if gp.considerPositions { v.Set{...} ) }

if qp := v.Encode(); qp != "" {
  uri += "?" + qp
}
```

Reply:
Does it make sense to include `consider_positions` in all basket request url, based on the input? Something like:
`uri := fmt.Sprintf("%s?consider_positions=%t", URIBasketMargins, ConsiderPositions)`
For ConsiderPositions = true input, this would produce:
`/margins/basket?consider_positions=true`
and for false:
`/margins/basket?consider_positions=false`
And we can have only one conditional statement to check for mode input and add it to url.
`if compactMode {
	uri += "&mode=compact"
}`
final url:
`/margins/basket?consider_positions=true&mode=compact`

Reply:
Makes sense, but I think it will be better to just use the stdlib net/url package to form this using `.Set` and `.Encode` instead. It seemed straightforward initially in our discussions but it really looks complex.

Reply:
Okay. Done.